std_msgs/Header header
string topic_name
string topic_arg
string stat
float32 resolution
uint8[] shape
uint8[] index
float32[] position
float32 arg_at_position
float32[] topic_map
